backend/src/main/java/com/huellapositiva/domain/repository/ESALContactPersonRepository.java

```diff
@@ -57,6 +57,9 @@ public Id save(ContactPerson contactPerson, com.huellapositiva.domain.model.valu
         JpaContactPerson jpaContactPerson = JpaContactPerson.builder()
                 .credential(jpaCredential)
                 .id(contactPerson.getId().getValue())
+                .name(contactPerson.getName())
+                .surname(contactPerson.getSurname())
+                .phoneNumber(contactPerson.getPhoneNumber())
```

Comment:
Sounds like a profile, what do you think?

Reply:
Sounds like a new table, this time is not a upsert because in the register these fields are mandatory.

Reply:
If you think the profile of a Contact Person will differ on the Volunteer's one, then it is reasonable to call for a new table.

Reply:
Yes, is absolutely different.

Reply:
It's done